pkg/reconciler/scheduling/edgeplacement/controller.go

```diff
@@ -305,7 +208,7 @@ func (c *controller) process(ctx context.Context, key string) error {
 	obj, err := c.edgePlacementLister.Cluster(clusterName).Get(name)
 	if err != nil {
 		if errors.IsNotFound(err) {
-			logger.Info("object deleted before handled")
+			logger.V(2).Info("object deleted before handled")
```

Comment:
Is this logger already at an elevated V level or does this statement actually correspond with `-v=2` ?  I think that this particular message is too inconsequential to appear at `-v=2`; I would make it 3 or a higher number.

Reply:
Just checked the logger, its `.level` field equals 0 before calling `V`. Changed to 3.